.gitignore

```
@@ -52,3 +52,4 @@ coverage.xml
 # Others
 *.DS_Store
 *.log
+*.txt
```

On garde les documents dde type .txt comme requirements. 
Enleve stp cette ligne :
```suggestion
*.txt
```